src/examples/tags/TagsColor.tsx

```diff
@@ -7,31 +7,31 @@
 
 import React from 'react';
 import { Tag } from '@zendeskgarden/react-tags';
-import { Row, Col } from '@zendeskgarden/react-grid';
+import { Grid } from '@zendeskgarden/react-grid';
 
 const Example = () => (
-  <Row>
-    <Col textAlign="center">
+  <Grid.Row>
+    <Grid.Col textAlign="center">
       <Tag hue="blue">
         <span>Conifer</span>
       </Tag>
-    </Col>
-    <Col textAlign="center">
-      <Tag hue="red">
+    </Grid.Col>
+    <Grid.Col textAlign="center">
+      <Tag hue="dangerHue">
```

Comment:
Nice, this demonstrates more range for the `hue` prop

Reply:
Hehe, yeah thats what I was hoping for!